packages/docusaurus-plugin-content-docs/package.json

```diff
@@ -5,6 +5,7 @@
   "main": "lib/index.js",
   "exports": {
     "./client": "./lib/client/index.js",
+    "./node": "./lib/node/index.js",
```

Comment:
If it's `client` I'd rather have its counterpart called `server`, not `node`...

Reply:
👍 

Considering we don't document yet `@docusaurus/plugin-content-docs/client` as public api, it's a good time to make a choice and commit to it.

What is better:
- client/server ?
- browser/node ?

Reply:
I like `client`/`server`. `browser` is inadequate because there's SSR... FWIW, I think we should change our tsconfig from `tsconfig.browser.json` to `tsconfig.client.json` as well